paasta_tools/contrib/rightsizer_soaconfigs_update.py

```diff
@@ -169,6 +169,34 @@ def get_recommendation_from_result(result, keys_to_apply):
     return rec
 
 
+def get_cassandra_recommendation_from_result(result, keys_to_apply):
+    rec = {}
+    for key in keys_to_apply:
+        val = result.get(key)
+        if not val or val == NULL:
+            continue
+        if key == "cpus":
+            rec["cpus"] = float(val)
+        elif key == "cpu_burst_percent":
+            rec["cpu_burst_percent"] = min(1, float(val))
```

Comment:
It can be 0 too right? Also you could drop the `min()` as it's already defined in https://github.com/Yelp/paasta/blob/934b1435f9a602b463d9bdab0a55097f84042e40/paasta_tools/cli/schemas/autotuned_defaults/cassandracluster_schema.json#L21